rel/overlay/etc/default.ini

```diff
@@ -318,12 +318,8 @@ compression_level = 8 ; from 1 (lowest, fastest) to 9 (highest, slowest), 0 to d
 compressible_types = text/*, application/javascript, application/json, application/xml
 
 [replicator]
-; minimum time between a replicator job restart (milliseconds)
-start_delay = 0
-; random splay time between a replicator job restart (milliseconds)
-start_splay = 0
-; Maximum replicaton retry count can be a non-negative integer or "infinity".
-max_replication_retry_count = 10
+; Random jitter applied on replication job startup (milliseconds)
+;startup_jitter = 5000
```

Comment:
There is a default in code to handle. Eunit does override it. However, the Javascript integration tests that's the problem, it takes too long and times out. So had to add an option to dev/run to override config values from command line.